src/Property.php

```diff
@@ -13,6 +13,7 @@ class Property
         'int' => 'integer',
         'bool' => 'boolean',
         'float' => 'double',
+        'immutable' => immutable::class
```

Comment:
Can you explain why this interface is needed?

Reply:
It isn't was something i forgot to remove.